### ISSUE TITLE
[Programmatic Access – Make Code – Extensions] Incorrect role is defined for 'Help' icon control under 'Extensions' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3691]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link or a button or whether it is a non interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users. 

**Note**: User credentials should NOT be included in the bug.

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls appearing under "Extensions" page
8. Verify that Incorrect role is defined for 'Help' icon control.

**Actual Result:**
Incorrect role is defined as 'Button' instead of 'link' for 'Help' icon control under 'Extensions' screen.

**Expected Result:**
Correct role should be define as 'link' instead of 'Button' for 'Help' icon control under 'Extensions' screen.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.1.2; #Win10-Edge(Chromium)

![WCAG4 1 2_Extensions_Incorrect role is defined as 'Button' instead of link on 'Help' icon control under 'Extensions' screen](https://user-images.githubusercontent.com/72871089/99035957-b8ca5a00-25a6-11eb-8456-7b8a86852715.jpg)
